tests/Microsoft.Identity.Test.LabInfrastructure/LabServiceParameters.cs

```diff
@@ -14,7 +14,8 @@ public enum FederationProvider
         Shibboleth,
         ADFSv2019,
         B2C,
-        Ping
+        Ping,
+        CIAM
```

Comment:
nit: Ciam

Reply:
It is mentioned in the docs as CIAM which I am assuming is an acronym for something. 
cc @bgavrilMS

Reply:
Ciam is fine.